charts/latest/csi-driver-nfs/values.yaml

```diff
@@ -1,7 +1,7 @@
 image:
     nfs:
-        repository: mcr.microsoft.com/k8s/csi/nfs-csi
```

Comment:
Images in k8s-staging are only kept for 30 days before being deleted. It's better to use a tagged version from k8s.gcr.io/sig-storage. If we plan to make a release soon, then I think it's fine to use a tag that doesn't exist yet.

Reply:
`gcr.io/k8s-staging-sig-storage/nfsplugin:canary` image is built every day, I think that's ok for master branch, user could always use this image built from master branch

Reply:
Can we call it "unstable" or "dev" to make it more obvious that these images are not officially released versions and they should not rely on this for production.

The canary images are rebuilt when a PR merges. There is a job that refreshes the image every week as well but if that job fails for some reason and we don't notice/fix it, the default image deletion policy will kick in.